frontend/src/store/index.js

```diff
@@ -55,8 +55,8 @@ import fromPairs from 'lodash/fromPairs'
 import isEqual from 'lodash/isEqual'
 import assign from 'lodash/assign'
 import forOwn from 'lodash/forOwn'
-import moment from 'moment-timezone'
 
+import moment from '@/utils/moment'
```

Comment:
is the comment in this file still valid or can it be removed?

https://github.com/gardener/dashboard/blob/8f997118d9b5608d6ea9af71b091e31d251c90d5/frontend/src/store/index.js#L86-L88

Reply:
I think we should remove this method completely and use `dayjs.tz.guess() || 'UTC'` directly. We do we check against the list of possible timezones. If it return something it should be valid location. Or do I miss something?
https://github.com/iamkun/dayjs/blob/dev/src/plugin/timezone/index.js#L143-L145

Reply:
I have move everything into the `@/utils/moment.js` file.